addons/web/static/src/views/list/list_renderer.js

```diff
@@ -113,6 +120,9 @@ export class ListRenderer extends Component {
 
         this.cellToFocus = null;
         this.activeRowId = null;
+        this.isEditableDebug =
```

Comment:
```suggestion
        this.hasOptionalOpenFormViewColumn =
```
other suggestion: getter, by consistency with the others?